src/chi/components/activity/activity.scss

```diff
@@ -0,0 +1,163 @@
+@import '_variables';
+@import '_mixins';
+
+$border: 0.0625rem;
+
+.m-activity {
+  font-size: $text;
+
+  &__day {
+    .m-activity__header {
+      border-bottom: 0.0625rem solid rgba(set-color(black), 0.12);
+      font-weight: 600;
+      padding: 0.5rem 0;
+    }
+
+    .m-activity__content {
+      .m-activity {
+        &__item {
+          align-items: start;
+          border-bottom: $border solid set-color(grey, 20);
+          flex-direction: row;
+          display: flex;
+          justify-content: start;
+          padding: 1.25rem 0;
+
+          & .a-avatar {
+            height: 3rem;
+            margin-right: 0.75rem;
+            width: 3rem;
+          }
+
+          .m-activity__start {
+            min-width: 9rem;
+
+            & div {
+              display: flex;
+              align-items: center;
+
+              & .a-icon {
+                color: set-color(grey, 50);
+                margin-right: 1.5rem;
+              }
+
+              & .a-avatar {
```

Comment:
@davidojedacl this class is not needed, please remove it. It is being inherited from l.26

Reply:
done